scripts/WFCORE-5261-common-conf.adoc

```diff
@@ -0,0 +1,102 @@
+= Add Common Script Configuration File
+:author:            James R. Perkins
+:email:             jperkins@redhat.com
+:toc:               left
+:icons:             font
+:idprefix:
+:idseparator:       -
+
+== Overview
+
+There is a desire to have a common script configuration file, `common.conf`. This will be called before the standalone
+or domain script configuration file. The other scripts will also invoke the common script configuration file. It allows
+things like common environment variables to be set by users regardless of the script being used.
```

Comment:
Can you please specify the scope of things that will be taken into account of this custom configuration file? Will it be only environment variables?

Reply:
User could really put anything in there that can be executed. From the description of the RFE it seems to mostly discuss environment variables. However, we can't really validate or control what is placed in there.